src/main/java/org/fundacionjala/pivotal/pages/project/PageLogin.java

```diff
@@ -66,7 +84,25 @@ public boolean checkPageLogin() {
      * @param username String.
      * @param password String.
      */
-    public void asLogin(final String username, final String password) {
+    public void signIn(final String username, final String password) {
+        try {
+            WebDriverWait shortWait = new WebDriverWait(driver, 5);
+            shortWait.until(ExpectedConditions.visibilityOf(txtUsername));
+            setFieldsLogin(username, password);
+        } catch (TimeoutException k) {
+            dashBoard.setButtonDashborad();
+        } catch (NoSuchElementException e) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Combine this catch with the one at line 92, which has the same body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2147)